app/models/application_dates.rb

```diff
@@ -16,7 +16,7 @@ def edit_by
   end
 
   def days_remaining_to_edit
-    (edit_by - Time.now.to_date).to_i
+    (edit_by.to_date - Time.now.to_date).to_i
```

Comment:
What is this object if not a date? Should `ApplicationDate` insulate us from this?

Reply:
I think I need to work on this...

But I think `edit_by` (and friends) would need to be a `Time` rather than a `Date` because the cutoff would be midnight (UK time).

Reply:
This opens up a can of worms around timezones. `end_date` is basically `(submitted at time + 5 working days).end_of_day_in_uk` (if that makes sense). I've put some notes on this commit https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/447/commits/f342915711431614a4d001dbd57167a4dcdd9e44 about what I think is the right thing to do here.